### DISCUSSION TITLE
Installing using CLI

### DISCUSSION THREAD

**Discussion by Nimaj1994** [ActivityWatch/activitywatch#1121] [Q&A] [answered]
Hi,
I have a project and part of it, includes ActivityWatch and I should be able to install ActivityWatch automatically during installation of my software. Simplest way is an option to install ActivityWatch using cmd. My target os is windows.
Thanks

**Comment by Bananeweizen** [comment, marked as answer]
Use chocolatey. It has a package for activitywatch.

**Comment by PhilippDehner** [comment]
> Use chocolatey. It has a package for activitywatch.

Or WinGet
